cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules-external.yaml

```diff
@@ -8,31 +8,35 @@ metadata:
   namespace: rook-ceph
 spec:
   groups:
-  - name: persistent-volume-alert.rules
-    rules:
-    - alert: PersistentVolumeUsageNearFull
-      annotations:
-        description: PVC {{ $labels.persistentvolumeclaim }} utilization has crossed
-          75%. Free up some space.
-        message: PVC {{ $labels.persistentvolumeclaim }} is nearing full. Data deletion
-          is required.
-        severity_level: warning
-        storage_type: ceph
-      expr: |
-        (kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"})) / (kubelet_volume_stats_capacity_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"})) > 0.75
-      for: 5s
-      labels:
-        severity: warning
-    - alert: PersistentVolumeUsageCritical
-      annotations:
-        description: PVC {{ $labels.persistentvolumeclaim }} utilization has crossed
-          85%. Free up some space immediately.
-        message: PVC {{ $labels.persistentvolumeclaim }} is critically full. Data
-          deletion is required.
-        severity_level: error
-        storage_type: ceph
-      expr: |
-        (kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"})) / (kubelet_volume_stats_capacity_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"})) > 0.85
-      for: 5s
-      labels:
-        severity: critical
+    - name: persistent-volume-alert.rules
```

Comment:
@travisn @subhamkrai This file is not a go-generated file. This file is generated by Mixin using Jsonnet.  This PR breaks the existing framework that is used to generate this file.
Can this PR be reverted and the https://github.com/rook/rook/tree/master/cluster/examples/kubernetes/ceph/monitoring directory be ignored permanently by this linter?

Reply:
Yes, let's revert this file and add it to the ignored list for the linter.

Reply:
@anmolsachan I'll do it by Monday. Thanks

Reply:
Thanks! Please do for both files prometheus-ceph-v14-rules.yaml and prometheus-ceph-v14-rules-external.yaml.

Reply:
#7402 @anmolsachan